ansible-ipi-install/roles/installer/tasks/main.yml

```diff
@@ -2,6 +2,7 @@
 # tasks file for installer
 - include_tasks: 10_get_oc.yml 
 - include_tasks: 20_extract_installer.yml
+- include_tasks: 25_customize_filesystem.yml
```

Comment:
Correct. I thought about adding some test logic to decide if we should run this, but I put on my ansible hat and tried to think and design idempotently instead of overloading with script logic. If the "fake" root directories are empty, then the filetranspile task will simply result in outputting unchanged ignition config files.